Spigot-Server-Patches/0463-Async-command-map-building.patch

```diff
@@ -0,0 +1,44 @@
+From 5ccaf0bce5b7928151762866ba6383aeaef4bd81 Mon Sep 17 00:00:00 2001
+From: Callahan <mr.callahhh@gmail.com>
+Date: Wed, 8 Apr 2020 02:42:14 -0500
+Subject: [PATCH] Async command map building
+
+
+diff --git a/src/main/java/net/minecraft/server/CommandDispatcher.java b/src/main/java/net/minecraft/server/CommandDispatcher.java
+index 37b1a7947..1a99e37bc 100644
+--- a/src/main/java/net/minecraft/server/CommandDispatcher.java
++++ b/src/main/java/net/minecraft/server/CommandDispatcher.java
+@@ -241,6 +241,14 @@ public class CommandDispatcher {
+         if ( org.spigotmc.SpigotConfig.tabComplete < 0 ) return; // Spigot
+         // CraftBukkit start
+         // Register Vanilla commands into builtRoot as before
++        // Paper start - Async command map building
++        java.util.concurrent.ForkJoinPool.commonPool().submit(() -> {
```

Comment:
Shouldn't this be `invoke` so its executed? You're just `submit`ting it to queue but never executing.